ui/src/routes/_layout/organizations/$orgId.products.$productId.repositories.$repoId.index.tsx

```diff
@@ -62,16 +62,20 @@ import {
 } from '@/components/ui/tooltip';
 import { useToast } from '@/components/ui/use-toast';
 import { ToastError } from '@/components/toast-error';
+import { getStatusBackgroundColor } from '@/helpers/getStatusColors';
+import { OrtRunJobStatus } from '@/components/ort-run-job-status';
 
 const RepoComponent = () => {
   const params = Route.useParams();
   const navigate = useNavigate();
   const { toast } = useToast();
+  const pollInterval =
+    Number.parseInt(import.meta.env.VITE_RUN_POLL_INTERVAL) || 10000;
```

Comment:
This should probably be outside of the component, so it doesn't try parsing the int on every render.